docker-compose.yaml

```diff
@@ -10,6 +10,15 @@ services:
       - "5432:5432"
     volumes:
       - postgres_db:/var/lib/postgresql/data
+  web:
+    container_name: gin-backend
```

Comment:
please do not use the framework for the container name. What if you have more than 1 project using the same framework?